### PR TITLE
chore: setup tokenization

### PR DESCRIPTION
**Pull request by HonkingGoose**
## Changes:

- Copy/paste @TWiStErRob's regex

## Context:

I first started by copy/pasting the example regex from the Material for MkDocs manual, and then @TWiStErRob had a better regex for us to try.

Read the [Material for MkDocs about `separator`](https://squidfunk.github.io/mkdocs-material/setup/setting-up-site-search/#built-in-search-plugin).

Helps with issue #264.

### CODE REVIEW

File: mkdocs.yml
Comment (viceice) [reply mkdocs.yml:179]: if this is better than before, go for it.

**Review by viceice — APPROVED** [review]

File: mkdocs.yml
Comment (TWiStErRob) [reply mkdocs.yml:179]: It is definitely better, at least there's a workaround now, in current prod there's nothing a user can do to get an exact match:
image
image
image

Comment (HonkingGoose) [reply mkdocs.yml:179]: Let's go with a partial fix for now. 😄

**Review by HonkingGoose — COMMENTED** [review]
I've tested the search in my Codespaces preview. Search work fine as far as I can tell. I have not seen anything that's obviously broken.

I'll let the maintainers review and eventually merge this. 😉

**Comment by rarkins** [comment]
How can we verify it's working as intended?

**Comment by TWiStErRob** [comment]
@rarkins expand the thread above, there are screenshots from before merge, if you get better results, it's working :)

Also in general just search for random things you can think of and if you can find it, it works.

**Comment by rarkins** [comment]
I happened to use the docs site for regular reasons and wonder if this update has broken some important search capabilities by tokenizing too much? For example:

image

image

image

**Comment by TWiStErRob** [comment]
Hmm, it looks like it needs to be able to split on *something*:

image

image

image

In your search `postu` is a word, but that word doesn't exist in the indexes.

**Comment by rarkins** [comment]
We might live with that if it weren't for the fact that an exact match returns no result!

**Comment by TWiStErRob** [comment]
Before this change, exact returned no result either: https://github.com/renovatebot/renovatebot.github.io/pull/265#discussion_r1073615075
Right now, only "exact lowercase" match is empty.
So I think you want to disable that part only, see https://github.com/renovatebot/renovatebot.github.io/issues/264#issuecomment-1375422381

**Comment by rarkins** [comment]
Oh, I didn't realize that! Guess I never tried it before, or that I only needed to type part of the word before clicking

**Comment by rarkins** [comment]
I'll close my PR for now

**Pull request merged**
